backend-app/controllers/authController.js

```diff
@@ -157,16 +157,28 @@ exports.protect = async (req, res, next) => {
     req.user = user;
```

Comment:
i guess req.user includes the password so the **getMe** controller should make the password property null then send the response.
@bellaabdelouahab

Reply:
ah yes i will fix it in next pr @muttaqin1